Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/InputActionsEditorWindowUtils.cs

```diff
@@ -14,12 +14,15 @@ internal class InputActionsEditorWindowUtils
         public static void SaveAsset(SerializedObject serializedAsset)
         {
             var asset = (InputActionAsset)serializedAsset.targetObject;
+
             // for the global actions asset: save differently (as it is a yaml file and not a json)
             if (asset.name == ProjectWideActionsAsset.kAssetName)
```

Comment:
Same comment as for drawer above, delegate check to InputSystem.IsProjectWideActionAsset(asset) or similar